packages/rancher-monitoring/generated-changes/patch/Chart.yaml.patch

```diff
@@ -16,7 +16,16 @@
  apiVersion: v2
  appVersion: 0.46.0
  dependencies:
-@@ -59,19 +68,17 @@
+@@ -14,7 +23,7 @@
+ - condition: hardenedKubelet.enabled
+   name: hardenedKubelet
+   repository: file://./charts/hardenedKubelet
+-- condition: hardenedNodeExporter.enabled
++- condition: nodeExporter.enabled
```

Comment:
I don't think we should change this condition from `hardenedNodeExporter.enabled` to `nodeExporter.enabled`

Reply:
Sure I can put it back.  The default value for hardenedNodeExporter is true; my thinking was to always enable the node exporter pushprox when the node exporter is enabled.  I can change that logic if you would prefer to have them separate.

Reply:
Let's keep them separate since the node-exporter flag is reserved for someone who wants to stick with the old approach (no additional pushproxes). This will be an opt-in flag to support deploying node-exporter in hardened clusters where even the nodeExporter port cannot be scraped.